package.json

```diff
@@ -1,7 +1,11 @@
 {
   "name": "coc-camel",
-  "version": "0.0.0-development",
```

Comment:
version 0.0.0 is already released on the npm registry. I think it is better to use an higher version, like 0.1.0

Reply:
Thank you. I have already included this PR changes in v0.0.0

Reply:
Should try to never release something which as not pushed to the repository.
I think that now we can let it go with 0.0.0 version and need to increment the version to 0.1.0 anyway

Reply:
I still think that you should increase the version number. the version that was published is not matching something in the repository so we should forget it, inlclude the required modifications and upgrade the version and then redo a release and create a correspondign tag.

Reply:
Thank you for explaining it. I believe I found where it was causing the errors. I see errors below when I run yarn install.
```
yarn install v1.22.19
[1/5] Validating package.json...
warning coc-camel@0.0.0-development: The engine "coc" appears to be invalid.
[2/5] Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

shell returned 1
```
Now I have updated the new yarn.lock file

Reply:
Thank you for the feedback. I have increased the version number and also updated the workflow file for providing autopublish and autotag feature. Please I'll work with you this afternoon to create a repository secret called `NPM_TOKEN` for enabling auto publish feature.